packages/fxa-auth-server/package.json

```diff
@@ -77,6 +77,7 @@
     "node-uap": "git+https://github.com/vladikoff/node-uap.git#9cdd16247",
     "nodemailer": "2.7.2",
     "otplib": "8.0.1",
+    "pem-jwk": "^2.0.0",
```

Comment:
Just noting my own though process, that we're already using this dep in fxa-auth-server, which is nice.